### PR TITLE
Synkron 276 Added getAllOrder with lineItems and getOrder with lineItems

### PR DESCRIPTION
**Pull request by Hardik2023103**

### CODE REVIEW

File: src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    public OrderResponse getAllOrdersWithLineItems(HashMap<String, String> params) {
+        URI uri = addParameters(new URI(sellingRegionEndpoint + "v3/merchants/" + accessCredentials.getMerchantId() + V3_ORDER_END_POINT), params);
+
+        HttpRequest request = get(uri);
+        HttpResponse.BodyHandler<OrderResponse> handler = new JsonBodyHandler<>(OrderResponse.class);
+        return getRequestWrapped(request, handler);
+    }
+
+    @SneakyThrows
+    public OrderElement getOrderWithLineItems(HashMap<String, String> params, String orderId) {
```
Comment (DhruvFefar2001) [inline src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java]: change method name "getOrderWithLineItems" to "getOrder"

File: src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import static io.github.dftrakesh.cloverrest.constantcode.ConstantCodes.V3_ORDER_END_POINT;
+
+public class CloverRestOrder extends CloverRestSDK {
+
+    public CloverRestOrder(AccessCredentials accessCredentials) {
+        super(accessCredentials);
+    }
+
+    @SneakyThrows
+    public OrderResponse getAllOrdersWithLineItems(HashMap<String, String> params) {
```
Comment (DhruvFefar2001) [inline src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java]: change method name "getAllOrdersWithLineItems" to getOrders

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/ListOrderLineItems.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package io.github.dftrakesh.cloverrest.model.Order;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ListOrderLineItems {
```
Comment (DhruvFefar2001) [inline src/main/java/io/github/dftrakesh/cloverrest/model/Order/ListOrderLineItems.java]: change class name "ListOrderLineItems" to "OrderLineItems"

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/ListOrderLineItems.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    private List<OrderLineItems> elements;
```
Comment (DhruvFefar2001) [inline src/main/java/io/github/dftrakesh/cloverrest/model/Order/ListOrderLineItems.java]: change name elements to orderLineItemList

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderElement.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    private String id;
+    private String href;
+    private String currency;
+    private String paymentState;
+    private Boolean taxRemoved;
+    private Boolean isVat;
+    private String state;
+    private Long createdTime;
+    private Long modifiedTime;
+    private ListOrderLineItems lineItems;
```
Comment (DhruvFefar2001) [inline src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderElement.java]: change name "lineItems" to orderLineItems

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderElements.java [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package io.github.dftrakesh.cloverrest.model.Order;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OrderElements {
+    private List<OrderElement> elements;
```
Comment (DhruvFefar2001) [inline src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderElements.java]: change name elements to orderElementList

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderLineItems.java [hunk 7]
```diff
@@ -0,0 +1,21 @@
+package io.github.dftrakesh.cloverrest.model.Order;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OrderLineItems {
```
Comment (DhruvFefar2001) [inline src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderLineItems.java:8]: change class name OrderLineItems to OrderLineItem

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderResponse.java [hunk 8]
```diff
@@ -0,0 +1,11 @@
+package io.github.dftrakesh.cloverrest.model.Order;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OrderResponse {
+    private List<OrderElement> elements;
```
Comment (DhruvFefar2001) [inline src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderResponse.java]: change name "elements" to "orderElementList"

File: src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+        URI uri = addParameters(new URI(sellingRegionEndpoint + "v3/merchants/" + accessCredentials.getMerchantId() + V3_ORDER_END_POINT), params);
```
Comment (DhruvFefar2001) [inline src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java]: create constants code "v3/merchants" variable
according to on getOrderWithLineItems method

**Review by DhruvFefar2001 — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java (hunk 1 above)
Comment (Hardik2023103) [reply src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java]: Changed the method name from "getOrderWithLineItems" to "getOrder"

File: src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java (hunk 2 above)
Comment (Hardik2023103) [reply src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java]: Changed the method name from "getAllOrdersWithLineItems" to getOrders

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/ListOrderLineItems.java (hunk 3 above)
Comment (Hardik2023103) [reply src/main/java/io/github/dftrakesh/cloverrest/model/Order/ListOrderLineItems.java]: Changed the class name to OrderLineItems

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/ListOrderLineItems.java (hunk 4 above)
Comment (Hardik2023103) [reply src/main/java/io/github/dftrakesh/cloverrest/model/Order/ListOrderLineItems.java]: changed the elements name to orderLineItemList

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderElement.java (hunk 5 above)
Comment (Hardik2023103) [reply src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderElement.java]: changed name "lineItems" to orderLineItems

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderElements.java (hunk 6 above)
Comment (Hardik2023103) [reply src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderElements.java]: Changed name elements to orderElementList

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderLineItems.java (hunk 7 above)
Comment (Hardik2023103) [reply src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderLineItems.java:8]: Changed  to OrderLineItem

File: src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java (hunk 9 above)
Comment (Hardik2023103) [reply src/main/java/io/github/dftrakesh/cloverrest/CloverRestOrder.java]: Changed "v3/merchants" to "V3_MERCHANTS" in both methods

File: src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderResponse.java (hunk 8 above)
Comment (Hardik2023103) [reply src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderResponse.java]: Changed name "elements" to "orderElementList"

**Review by DhruvFefar2001 — APPROVED** [review]

**Review by DhruvFefar2001 — APPROVED** [review]

**Pull request merged**